src/forklift/cli.py

```diff
@@ -379,3 +387,29 @@ def speedtest(pallet_location):
     print('{1}Dry Run Output{0}{2}{3}'.format(Fore.RESET, Fore.CYAN, linesep, dry_report))
     print('{1}Repeat Run Output{0}{2}{3}'.format(Fore.RESET, Fore.CYAN, linesep, repeat_report))
     print('{3}{0}{1}Speed Test Results{3}{0}{2}Dry Run:{0} {4}{3}{2}Repeat:{0} {5}'.format(Fore.RESET, Fore.GREEN, Fore.CYAN, linesep, dry_run, repeat))
+
+
+def update_static(file_path):
+    log.info('updating/creating static data for pallet(s) in %s', file_path)
+
+    start_seconds = clock()
+
+    git_errors = git_update()
```

Comment:
`update_static` is only run when invoked via the cli. I thought that it would be good to through in the `git_update` call to make sure that everything's up-to-date. But maybe that is assuming too much?